lib/Selenium/Remote/Driver/ActionChains.pm

```diff
@@ -0,0 +1,155 @@
+package Selenium::Remote::Driver::ActionChains; 
+# ABSTRACT: Action chains for Selenium::Remote::Driver
+use Moo; 
+
+has 'driver' => ( 
+    is => 'rw', 
```

Comment:
perhaps we should 'ro' this and just make new ::ActionChains objects? avoiding state when possible and such